contrib/get_repository_tokens.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python
+
+import requests
+import re
+import json
+import sys
+import getopt
+ 
+ 
+s = requests.session()
+
+def get_rels(response):
+    links = response["link"].split(',')
+    regex = re.compile(r'<(.*?)>; rel="(\w+)"')
+    rels_hash = {}
+    for link in links:
+        href, name = regex.findall(link)[0]
+        rels_hash[name] = href
+    return rels_hash
+                        
+
+def get_token(match_repo_url):
+    resp = s.get("https://scc.suse.com/connect/organizations/repositories", headers={'Accept':'application/vnd.scc.suse.com.v4+json','cache-control':'no-cache'})
+    repositories = []
+
+    while True:
+        for repository in resp.json():
+            if match_repo_url in repository['url'].split("?")[0]:
+                print repository['url'].split("?")[0]+","+repository['url'].split("?")[-1]
+        rels = get_rels(resp.headers)
+        if not 'next' in rels:
+            break
+        resp = s.get(rels['next'])
+
+def print_USAGE():
+    print 'Usage:'
+    print '\t'+self+' -u <username> -o <password> <part of repository URL> [... <part of repository URL>]'
```

Comment:
**Python2 EOL**

Usage instruction is wrong. The password flag needs to be `-p`.

```suggestion
    print('\t'+self+' -u <username> -p <password> <part of repository URL> [... <part of repository URL>]')
```